### ISSUE TITLE
Can I publish an article about this please?

### ISSUE THREAD

**Issue by ON7CFI** [luickk/noaa_satellite_ground_station#3]
Dear Luick,
I have written an article on how to install and use your NOAAStation script. Doing that, I have amended (dare I say: improved?) your script. Now I want to publish that article along with the amended code in CQ-QSO, the bimonthly magazine of the UBA (the Belgian association of radioamateurs). Can I get your approval for this, please?
Thanks and very best regards,
Jantje.
ON7CFI
on7cfi@uba.be

**Comment by luickk** [comment]
Replied to your email but never to this issue. I will and did certainly allow it :)

**Comment by ON7CFI** [comment]
Dear Luick,

 

Yes, I did indeed receive your e-mail confirming your permission. That was already a while ago and indeed quite quickly after I sent you my question.

 

The issue ticket on github predates that e-mail, because I found out your e-mail address only afterwards.

 

The article in the meantime has been published and has brought at least one person to be interested in satellite reception. Your effort (and mine, dare I say) has not been invain.

 

Thanks again,

 

Jantje.
